app/src/main/java/org/sopt/and/domain/model/User.kt

```diff
@@ -0,0 +1,7 @@
+package org.sopt.and.domain.model
+
+data class User(
+    val username: String,
+    val password: String,
+    val hobby: String,
```

Comment:
이 부분은 확장성을 고려한다면 hobby 클래스로 받는게 좋아보여요
예시로는 다음과 같이요 
```suggestion
    val hobby: Hobby,
```
1번 질문에 대한 제 생각으로는  hobby라는 클래스를 따로 두는 것은 좋은거 같아요. 

추후 취향을 반영할 때 취향에 대한 이름 뿐만 아니라 여러가지 요소가 들어가는 방향으로 진행된다면 hobby data class 를 변경하고 유저에 따라 취향에 관한 것을 한번에 관리할 수 있을 거 같네요